test/rdoc/rdoc_markup_to_html_crossref_test.rb

```diff
@@ -33,6 +33,33 @@ def test_convert_CROSSREF
     assert_equal para("<code># :stopdoc:</code>:"), result
   end
 
+  def test_convert_CROSSREF_backslash_in_tt
+    hyperlink_all = @options.hyperlink_all
+    @options.hyperlink_all = false
+
+    formatter = RDoc::Markup::ToHtmlCrossref.new @options, 'C9.html', @c9_b
+    result = formatter.convert '<tt>C1</tt>'
+
```

Comment:
How about removing this empty line because `format = ...; result = ...; assert_equal ...` are related?

```suggestion
```

Reply:
I've reduced the lines this test takes